### PR TITLE
Helpers to make views a little more like UIKit

### PR DESCRIPTION
**Pull request by gabriel**

### CODE REVIEW

**Comment by johnboiles** [comment]
Don't feel strongly, but these feel on the fringes of the scope of YOLayout.

File: YOLayout/AppKit/YONSView.h [hunk 1]
```diff
@@ -59,4 +59,12 @@
  */
 - (void)layoutView;
 
+//
+// Helpers to make views a little more like UIKit
+//
+
+@property BOOL clipToBounds; // Writable alias for wantsDefaultClipping
+@property NSInteger flag; // Instead of tag (writable)
```
Comment (johnboiles) [inline YOLayout/AppKit/YONSView.h]: This property in particular feels kinda out there

**Comment by johnboiles** [comment]
UIKit-ifying NSView could probably be a library in itself. Reminds me of https://github.com/BigZaphod/Chameleon

**Comment by gabriel** [comment]
Yeah I agree.. I'll nuke this

**Pull request closed without merging**